### PR TITLE
Whitelist European Space Agency

### PR DESCRIPTION
**Pull request by izcoser**
The Open Access at ESA website is legit, from the European Space Agency. Should not be blocked.

https://open.esa.int

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -45773,4 +45774,4 @@
     "layerzeronetwork.xyz",
     "layerzeroprotocols.com"
   ]
-}
+}
```
Comment (legobeat) [inline src/config.json]: @izcoser Thank you for your contribution! Would you be unable to undo the change on this line?

Comment (izcoser) [reply src/config.json]: Sure. Done.

**Review by legobeat — APPROVED** [review]
